### PR TITLE
Ap286 remove social security claim

### PR DESCRIPTION
**Pull request by pauldougan**
[AP-286](https://govukverify.atlassian.net/browse/AP-286)
## Why

No longer required

## What

Remove https://vocab.account.gov.uk/v1/socialSecurityRecord claim from examples and documentations

## Technical writer support

Do you need a tech writer's support, for example to review your PR? - **YES**

## How to review

- check out the branch 
- preview locally using docker script
- ensure there are no mentions of the https://vocab.account.gov.uk/v1/socialSecurityRecord claim

## Changelog

thus pr contains an update to the changelog

## Confirm

- [x] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [xx] Where there is any overlap I have updated or opened a PR for corresponding changes

[AP-286]: https://govukverify.atlassian.net/browse/AP-286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/partials/_changelog.erb [hunk 1]
```diff
@@ -4,10 +4,11 @@
...
-| May 17 2024       | New guidance "<a href="/test-your-integration/build-mocks/#build-mocks-to-work-with-gov-uk-one-login">Build mocks to work with GOV.UK One Login</a>" to explain how to build mocks as a part of testing your service.                                                                                                   |
+| Jul 3 2024       | Remove the `https://vocab.account.gov.uk/v1/socialSecurityRecord` claim                                                                                                     |
+| Jun 21 2024      | Updated guidance <a href="/integrate-with-integration-environment/authenticate-your-user/#error-handling-for-make-a-request-to-the-authorize-endpoint">"Error handling for ‘Make a request to the /authorize endpoint"</a> to clarify the <code>{"message": "Internal server error"}</code>HTTP 502 Bad gateway error.                                                                                                   |
```
Comment (PippaClarkGDS) [inline source/partials/_changelog.erb:8]: we're mixing tenses here ... should it be past or present?

**Review by philf999 — APPROVED** [review]
Looks good to me

File: source/partials/_changelog.erb (hunk 1 above)
Comment (pauldougan) [reply source/partials/_changelog.erb:8]: good point let me bring it in line with the rest of the changelog

Comment (pauldougan) [reply source/partials/_changelog.erb:8]: I suspect the changelog needs looking at as a whole, I merely mimicked the earlier wording

**Review by PippaClarkGDS — APPROVED** [review]

**Pull request merged**